templates/contacts/contact_invite_participants.haml

```diff
@@ -132,7 +132,12 @@
                               {% short_datetime object.created_on %}
 
                           %td.invite-column.value-received.field_received
-                            %a.invite.button.button-primary{href:"{% url 'contacts.contact_invite_participants' %}", 'data-uuid': "{{object.uuid}}", 'status': "active"}
+                            %a.invite.button.button-primary(
```

Comment:
I think we can have the `has_no_connection` statement like this (rather than having the `disconnected` class doing the functionality of hiding):

```
{% if not object|has_no_connection %}
    %a.invite.button.button-primary(
        href="{% url 'contacts.contact_invite_participants' %}"
        data-uuid="{{object.uuid}}"
        status="active"
    )
{% endif %}
```

Reply:
Updated